sdk/servicebus/azure-servicebus/swagger/servicebus-swagger.json

```diff
@@ -514,17 +749,21 @@
             "schema":{
               "type": "object"
             }
+          },
+          "default": {
+            "$ref": "#/responses/ServiceBusManagementErrorResponse"
           }
         }
       }
     },
-    "/$Resources/queues": {
+    "/$Resources/{entityType}": {
       "get": {
-        "operationId": "Queues_Get",
-        "summary": "Get Queues",
-        "description": "Get the details about the Queues of the given ServiceBus namespace.",
+        "operationId": "Gets a list of all the entities in Service Bus.",
```

Comment:
Better make this shorter. The generated method name in Python is `gets_a_list_of_all_the_entities_in_service_bus`

Reply:
Oh. I thought I had it as a description. Lol